# Request (empty)
---
# Response
string[] object_names
geometry_msgs/Point[] object_positions